components/home/Banner.vue

```diff
@@ -0,0 +1,73 @@
+<script setup lang="ts">
+  import { DateStringObject } from 'models/DateTime'
+
+  const props = defineProps<{
+    content: {
+      caption: string
+      date: {
+        start: string
+        end: string
+      }
+      image: string
+      title: string
+    }
+  }>()
+
+  const { mobile } = useDisplay()
+
+  // Convert event dates into readable and usable format
+  const { content } = toRefs(props)
+  const startDate: DateStringObject = dateStringToStringObject(
```

Comment:
Could use a computed property here to preserve reactivity, but I guess it's not really necessary.